pkg/logging/logger.go

```diff
@@ -31,21 +32,45 @@ func init() {
 func SetLevel(level string) {
 	var err error
 	zerolog.ErrorStackMarshaler = pkgerrors.MarshalStack
-
 	logLevel, err := zerolog.ParseLevel(level)
 	if err != nil || level == "" { // allow turning off logging
 		logLevel = zerolog.Disabled
 	}
 
+	// "Writer" is just the the anonymous field "io.Writer" in the  struct
 	var output io.Writer = zerolog.ConsoleWriter{
 		Out:          os.Stdout,
 		TimeFormat:   LOG_TIME_FORMAT,
 		TimeLocation: time.Local,
 	}
+	consoleWriterLeveled := &LevelWriter{Writer: output, Level: logLevel}
```

Comment:
This is an unnecessary constraint. Daemon should simply output to stdout as before, and let the caller handle log storage.

Reply:
This is just for having multiple log levels outputs one to stdout one to a file with debug/trace level logs.

This allows us to take a look at the logs while keeping pod logs clean, also added lumberjack so as to ensure the log file is rotated and doesn't ever get too big.

Reply:
systemd should take care of it, or whatever init service being used. For systemd, we can always use syslog to or `journalctl -u cedana.service` to fetch logs. This should not be baked into the binary.